### DISCUSSION TITLE
24-01-2024 Steering Committee Meeting Minutes and Resolution

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#72] [Steering Committee]
1. Following communications with Jesse, we've agreed to appoint him as the new Community Manager. This role will include an initial period of adjustment and a probationary phase.

2. Regarding the Growth Manager role, we recognized that the original requirements, which emphasized development and technical skills and developer-focused tasks, might not align perfectly with PartnershipDude's background. Consequently, we've decided to revise and update the job description, renaming it to Community Growth Manager. The revised role will focus on member recruitment, business development, and supporting Darwinia in broadening its ecosystem and facilitating communications with application developers. We believe that PartnershipDude, with his background, is a more suitable fit for this updated role. Ran will be in touch with PartnershipDude to discuss and confirm the finer details of this Community Growth Manager position.

3. It's agreed that the budget for the manager positions should be disclosed during the proposal voting process. The specific remuneration executed will not exceed this budget. However, we recommend that the details of the compensation can be kept confidential, disclosed only to the Steering Committee.

Ref:
https://github.com/orgs/dcdao/discussions/53

**Comment by 2xJDubs** [comment]
I do think we still need someone with technical expertise to interface with prospective dapp developers and projects to help them realize the capabilities that can be achieved with Darwinia's tech.  I think having someone to help projects realize the potential will be very beneficial for adoption.  Would Aki be the one who would facilitate this kind of role or is there someone else that can fill this role?

**Comment by Laki389** [comment]
>     2. Regarding the Growth Manager role, we recognized that the original requirements, which emphasized development and technical skills and developer-focused tasks, might not align perfectly with PartnershipDude's background. Consequently, we've decided to revise and update the job description, renaming it to Community Growth Manager. The revised role will focus on member recruitment, business development, and supporting Darwinia in broadening its ecosystem and facilitating communications with application developers. We believe that PartnershipDude, with his background, is a more suitable fit for this updated role. Ran will be in touch with PartnershipDude to discuss and confirm the finer details of this Community Growth Manager position.



To be honest, as much as I sincerely hoped that the new position of Growth Manager would bring new, positive winds to DCDAO and Darwinia, I was so disappointed that the Steering Committee had the opinion that a 19 year old Medical student would be suitable for this position.

I sincerely hoped that someone like Megan would be brought in, because I know about Megan in particular that she is probably doing great at Astar and is well paid there...

Were there other candidates?
Is there anyone from Itering who can cover this position?
Perhaps specialist websites for WEB3 jobs could help?
I also know that there is a job posting section on Polkadot's official Discord, so maybe a more suitable candidate could have been found?
Considering that $3000 a month is not a small amount?
Living in Eastern Europe, I'm sure that for that amount you can find someone with far more work and life experience and more suitable qualifications?

Becoming a good doctor means 14 hours a day listening to lectures and studying, 3 hours for everything else (eating, transportation, no RL) and 7 hours of sleep to have a good concentration during the day.

Considering the Growth Manager position is full-time, how will these things align, unless it's a new Vitalik, perhaps?




>     3. It's agreed that the budget for the manager positions should be disclosed during the proposal voting process. The specific remuneration executed will not exceed this budget. However, we recommend that the details of the compensation can be kept confidential, disclosed only to the Steering Committee.
> 
> 
> Ref: https://github.com/orgs/dcdao/discussions/53


I think there is no need for any secrecy for the following reasons:

- Payments are onchain, therefore they cannot be secret, the DCDAO Treasury address is known.

- In this vote at https://darwinia2.subsquare.io/democracy/referenda/15 all sums can be seen.

<img width="584" alt="3681701072101-pic" src="https://github.com/dcdao/discussion/assets/133054568/d8ac0186-82dd-45de-8062-5b025b5fe8f3">




- Since the salaries are sponsored by Darwinia Treasury, that means by all of us, they should be public.
